src/AppInstallerRepositoryCore/Microsoft/PinningIndex.cpp

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "PinningIndex.h"
+#include "SQLiteStorageBase.h"
+#include "Schema/Pinning_1_0/PinningIndexInterface.h"
+
+namespace AppInstaller::Repository::Microsoft
+{
+    PinningIndex PinningIndex::CreateNew(const std::string& filePath, Schema::Version version)
+    {
+        AICLI_LOG(Repo, Info, << "Creating new Pinning Index [" << version << "] at '" << filePath << "'");

```

Comment:
Are the logs for the pinning index meant to be Repo, or should they be SQL?

Reply:
I have to admit I didn't think of that, I just copied what the Portable index did. It looks like SQL is used exclusively at the level of the SQLiteWrapper, so I think Repo is fine here.